software/nosql/src/main/java/brooklyn/entity/nosql/couchbase/CouchbaseClusterImpl.java

```diff
@@ -229,8 +229,13 @@ protected void doStart() {
                 }
                 
                 ((CouchbaseNode)getPrimaryNode()).rebalance();
-            } else if (getQuorumSize()>1) {
-                log.warn(this+" is not quorate; will likely fail later, but proceeding for now");
+            } else {
+                if (getQuorumSize()>1) {
```

Comment:
it's long-winded but i think logic is correct.  we've established there are `upNodes` aka `serversToAdd`, and if we are in this `else` block then also either we're not quorate or we are only adding a single server.  in the latter case we might be quorate hence this check before warning.